stable/database/templates/service.yaml

```diff
@@ -20,15 +23,15 @@ metadata:
     domain: {{ .Values.admin.domain }}
     chart: {{ template "database.chart" . }}
     release: {{ .Release.Name | quote }}
-  name: {{ .Values.database.name }}-{{ default .Values.admin.serviceSuffix.balancer .Values.database.serviceSuffix.balancer }}
+  name: {{ template "database.fullname" . }}-{{ default .Values.admin.serviceSuffix.balancer .Values.database.serviceSuffix.balancer }}
```

Comment:
Same comment about default service name when it is a nodeport

Reply:
Will fix.